mergify_engine/subscription.py

```diff
@@ -69,11 +69,20 @@ def missing_feature_reason(owner):
 
     @classmethod
     def from_dict(cls, owner_id, sub):
+        # FIXME(sileht): Remove me in Janvier 2021, we should not support two kind of
+        # payload, we have wrongly token the token as dict instead of string, so we have
+        # to handle for a while, in the meantime the dashboard will be updated to return
+        # only the access_token.
+        tokens = dict(
+            (login, token["access_token"] if isinstance(token, dict) else token)
+            for login, token in sub["tokens"].items()
+        )
```

Comment:
```suggestion
        # FIXME(sileht): Remove me in January 2021, we should not support two kinds of
        # payload, we have wrongly sent the token as dict instead of string, so we have
        # to handle it for a while, in the meantime the dashboard will be updated to return
        # only the access_token.
        tokens = dict(
            (login, token["access_token"] if isinstance(token, dict) else token)
            for login, token in sub["tokens"].items()
        )
```
The tokens are only cached for one hour, so I don't think we need to wait that long, right?

Reply:
I just put a random date.